index.bs

```diff
@@ -837,11 +837,11 @@ urlPrefix: https://w3c.github.io/FileAPI/#dfn-; type: dfn;
 
 					1. Abort these steps.
 
-				1. Let |data| be a copy of the [=system clipboard data=] represented as [=clipboard items=]. For the MIME types defined in the [=mandatory data types=] list, |data| contains the sanitized copy of text/plain format.
+				1. Let |data| be a copy of the [=system clipboard data=] represented as [=clipboard items=]. For the MIME types defined in the [=mandatory data types=] list, |data| contains the content of text/plain format.
```

Comment:
Suggest that we change:

>For the MIME types defined in the [=mandatory data types=] list, |data| contains the content of text/plain format.

to 

For the MIME types defined in the [=mandatory data types=] list, |data| MAY be sanitized.

Reply:
Done.